app/uk/gov/hmrc/platopsgithubproxy/connector/GitHubConnector.scala

```diff
@@ -59,4 +64,64 @@ class GitHubConnector @Inject()(
           .map(entry => entry._1 + "=" + entry._2.mkString(","))
           .mkString("&")
   }
+
+  private implicit val hc = HeaderCarrier()
```

Comment:
Can this be moved to the scheduler, and be passed as an implicit param to `getRateLimitMetrics`?